tests/webdriver-e2e/page-objects/common.page.js

```diff
@@ -0,0 +1,18 @@
+class CommonPage {
+  get hamburgerMenu() { return $('#header-dropdown-link'); }
+  get logoutButton() { return $('.fa-power-off'); }
+  get modalBody() { return $('div.modal-body'); }
+  get yesButton() { return $('a.btn.submit.btn-danger'); }
+  // const modal = $('div.modal-dialog');
+  async logout() {
+    await (await this.hamburgerMenu).click();
```

Comment:
This is funny. We should definitely add a rule of how many chained awaits we can have :))

Reply:
yeah, I agree. It felt weird.